client/ayon_traypublisher/plugins/create/create_csv_ingest.py

```diff
@@ -644,9 +644,21 @@ def _add_representation(
         # convert ### string in file name to %03d
         # this is for correct frame range validation
         # example: file.###.exr -> file.%03d.exr
+        file_name = basename.split(".")[0]
         if "#" in basename:
             padding = len(basename.split("#")) - 1
-            basename = basename.replace("#" * padding, f"%0{padding}d")
+            seq_padding = f"%0{padding}d"
+            basename = basename.replace("#" * padding, seq_padding)
+            file_name = basename.split(seq_padding)[0]
+            is_sequence = True
+        if "%" in basename:
+            pattern = re.compile(r"%\d+d|%d")
+            padding = pattern.findall(basename)
+            if not padding:
+                raise CreatorError(
+                    f"File sequence padding not found in '{basename}'."
+                )
+            file_name = basename.split("%")[0]
```

Comment:
Is it correct that `file_name` ends up being the file without extension?

Reply:
Yeah in this case basename is the name with extension. I am assuming I should have name this file_head, right?